src/components/transactions/TxSigners/index.tsx

```diff
@@ -127,10 +129,18 @@ export const TxSigners = ({ txDetails, txSummary }: TxSignersProps): ReactElemen
   const canExecute = wallet?.address ? isExecutable(txSummary, wallet.address, safe) : false
   const confirmationsNeeded = confirmationsRequired - confirmations.length
   const isConfirmed = confirmationsNeeded <= 0 || canExecute
+  const showExecutor = executor || !isConfirmed
 
   return (
     <>
-      <List className={css.signers}>
+      <List
+        className={css.signers}
+        sx={{
+          '::before': {
+            height: `calc(100% - ${showExecutor ? '80px' : '40px'})`,
```

Comment:
Why so complicated? Can't the executor 🪓 just go directly after everything above?

Reply:
If we don't have conditional `height`, the stepper line bleeds into the executor positioning:

![image](https://user-images.githubusercontent.com/20442784/213716254-cff6a688-4945-4956-b8cf-946c3d133240.png)

vs.

![image](https://user-images.githubusercontent.com/20442784/213716430-23020945-ac13-43b1-82c9-01692a44b102.png)

Reply:
🤦‍♂️ I see what you mean now. Let me adjust it.